### DISCUSSION TITLE
Impact of sea-salt tuning on 20th century

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/cesm_dev#4] [2. Model bias]
The objective was to determine whether tuning sea-salt has an impact on 20th-century warming compared to what was observed in configuration 98b (https://github.com/NCAR/amwg_dev/issues/556)

We conducted a 20th-century simulation using configuration 104 (https://github.com/NCAR/amwg_dev/issues/575).

Configuration 104 has a more realistic sea-salt value compared to 98b:

BURDENSEASALT (kg/m2) 
- 98b => 3.23e-05 
- 104 => 1.13e-05

Unfortunately, the tuning does not significantly impact the 20th-century warming.

![Screen Shot 2024-09-04 at 11 02 01 AM](https://github.com/user-attachments/assets/fffddfd9-f82b-49f5-9f6d-db3c42910c2c)

@cecilehannay, @PeterHjortLauritzen, @adamrher, @brianpm, @JulioTBacmeister, @swrneale, @tilmes, @dlawrenncar, @islasimpson, @cmcclus

**Comment by adamrher** [comment]
@islasimpson can you point me to the LENS output so I can make a zonal mean -- time difference plot with LENS?

**Reply by islasimpson** [reply]
It's here: /glade/campaign/cesm/collections/CESM2-LE.  I'll try to update the various plots I made in the last round later.

**Comment by islasimpson** [comment]
Here are some plots.  Recall that last time we thought the lack of SWCF decline over the 20th century might be leading to the lack of warming in the historical.  Consistent with the fact that the historical temperature evolution hasn't changed much in 104, neither has the SWCF.  The SWCF is still not declining as much over the historical record as it was in LENS1.

![Screenshot from 2024-09-04 21-30-42](https://github.com/user-attachments/assets/70fde895-4ea2-4283-9452-9e258561185a)

Just like we had seen in 098b, we don't have as big of an increase in TGCLDLWP as we did in LENS2.

![Screenshot from 2024-09-04 21-33-19](https://github.com/user-attachments/assets/be3c08b6-2201-49a8-ac02-b3f69761de9b)

They still differ in the climatology as well...

![Screenshot from 2024-09-04 21-34-34](https://github.com/user-attachments/assets/3a4afe15-182e-4a26-beb4-5b9876303604)

The cloud droplet number concentration is still not increasing as much as in LENS2 and looks pretty close to 098b and the climatological CDNUMC is still high

![Screenshot from 2024-09-04 21-35-05](https://github.com/user-attachments/assets/9bfcde2e-29d0-47d6-bc9d-6080f1cc015b)

 Our thinking before was that the high CDNUMC was preventing CDNUMC from increasing as much as it used to and I think that thinking still holds, but the sea salt does not seem to be the primary cause of our higher CDNUMC than what we had in CESM2.

AOD is lower than it was in 098b, I think closer to observed values.  Lower than LENS2 [This paper](https://www.sciencedirect.com/science/article/pii/S1352231014003252) says global mean AOD in the 2000's was 0.126.

![Screenshot from 2024-09-04 21-37-48](https://github.com/user-attachments/assets/c01aa894-fc85-462f-b8cb-9feb5a193da9)

Here are the burdens.  Sea salt has lowered as expected.  The dust burden has changed a lot - not sure if that's expected.

![Screenshot from 2024-09-04 21-40-25](https://github.com/user-attachments/assets/394c8e64-defe-42a4-8df0-8f89670fb62f)

Overall, it seems we still need to understand why CDNUMC is so much higher as it doesn't seem to be because of the salt.

**Reply by cecilehannay** [reply]
Would it be possible to add run **92** (https://github.com/NCAR/amwg_dev/issues/543) to your SWCF plots? 
(92 didn't have RRTMGP)

**Reply by islasimpson** [reply]
Here are all the same plots as above with 092 added.  

SWCF:

<img width="750" alt="Screen Shot 2024-09-05 at 1 14 46 PM" src="https://github.com/user-attachments/assets/cafb3631-0a7c-4f32-8400-daddc7bebaed">

Cloud ice and liquid water paths

<img width="1066" alt="Screen Shot 2024-09-05 at 1 22 07 PM" src="https://github.com/user-attachments/assets/c5e30f27-3ba9-43e1-91ee-318b8d34d061">

CDNUMC:

<img width="1023" alt="Screen Shot 2024-09-05 at 1 24 40 PM" src="https://github.com/user-attachments/assets/12c3979c-c4ff-4587-b3a1-ad9fd57d87bb">

Burdens:

<img width="1138" alt="Screen Shot 2024-09-05 at 1 41 21 PM" src="https://github.com/user-attachments/assets/b7ad51ef-e264-470c-9355-1899385d8601">

This demonstrates how the dust in 104 is close to that in 92 but pretty different from 98.  The same goes for BURDENSOA.

**Reply by cecilehannay** [reply]
I believe the differences in `burdendust` and `burdeseasalt` (and maybe `burdensoa`) in 98 are expected.
They were coming from a bug in that tag and 98 that used the "wrong" optical properties for the **coarse mode (mode 3)**


```92: RRTMG: mam4_mode3_rrtmg_aeronetdust_c141106.nc```
```98b: RRTMGP: mam4_mode3_rrtmg_aeronetdust_sig1.2_dgnl.40_c150219.nc```  (**buggy**)
```104: RRTMGP: mam4_mode3_rrtmg_aeronetdust_c141106.nc```

**Reply by islasimpson** [reply]
I'm a bit confused.  I forgot I had made this plot in the past where in 102 the CDNUMC was brought way down.  This had a seasalt_emis_scale of 0.75 as opposed to 1.5 in the earlier runs.  104 also has seasalt_emis_scale of 0.75 but has a very different CDNUMC - it has gone back up again.  Something else must have changed between 102 and 104 other than the seasalt_emis_scale to bring CDNUMC back up

<img width="365" alt="Screen Shot 2024-09-06 at 8 48 42 AM" src="https://github.com/user-attachments/assets/1e02569a-4879-4398-86fd-9f9a9cc78377">

**Reply by islasimpson** [reply]
Here's an update of that with 104 added.

<img width="369" alt="Screen Shot 2024-09-06 at 8 56 51 AM" src="https://github.com/user-attachments/assets/6633dbdd-b8fe-4af2-a9ed-271d7849368e">

**Reply by adamrher** [reply]
I noticed that too, that the 104 drop number wasn't as low as the original run with reduced sea salt emissions. Very odd.

**Reply by tilmes** [reply]
@adamrher just catching up. Could it be changed with the tuning you imposed? What values have you changed there?

**Reply by adamrher** [reply]
Yes, the c8 tuning is responsible for the increased drop number in 104. 103 reverts the c8 tuning, and has the lower drop number as in 102 (which uses an older code base than 103, 104).

**Comment by cecilehannay** [comment]
@brianpm requested to add 92 to the TS historical timeseries. 
- 92: use RRTMG 
- 98b and 104 use RRTMGP

While we didn't suspect RRTMGP to be the culprit, we wanted  to confirm the issues with the TS warming is similar with/without RRTMGP. 

![Screen Shot 2024-09-05 at 9 54 54 AM](https://github.com/user-attachments/assets/fc2e2bf5-48a4-4454-a3ca-8d2741ce24f5)

**Reply by brianpm** [reply]
Kind of relief (for me) that the radiation scheme doesn't seem to be playing a factor. I had started to get worried that a slight inconsistency in the radiation schemes band boundaries and the cloud optical properties boundaries might be doing something. Seems to not be the case. Phew!

**Reply by islasimpson** [reply]
Following suggestions from @brianpm I'm adding some plots of a few more cloud related things...

CLDTOT is also pretty different between the runs.  It is decreasing more in the new runs prior to 1980.  In contrast, in the CESM1, it remained fairly constant until about 1980 and then declined dramatically.  CLDTOT is highest in 104 but it doesn't seem like there's much of a relationship between the starting value of CLDTOT and how much it declines.

<img width="1178" alt="Screen Shot 2024-09-11 at 5 47 40 PM" src="https://github.com/user-attachments/assets/d57c6160-ed19-463b-9deb-c4c6534335cb">

ACTREL (cloud top droplet effective radius) can also be compared with CESM2.  The radius is quite a bit smaller in the new runs compared to LENS2 although 104 is closer to LENS2 than the 092 and 098.

<img width="1177" alt="Screen Shot 2024-09-11 at 5 49 11 PM" src="https://github.com/user-attachments/assets/95c5ebd3-76f3-4c96-a5c5-bbfa24102f42">

Here are the other variables suggested by @brianpm but unfortunately we don't have CESM2 to compare with

<img width="1182" alt="Screen Shot 2024-09-11 at 5 55 35 PM" src="https://github.com/user-attachments/assets/99ab0553-4624-4f1a-bdbe-450523c1169e">

**Comment by adamrher** [comment]
I had been wondering if MOM6 is itself just resilient to cooling during the post war period. It's always difficult to figure out if it's a cloud bias impacting the SSTs, or vice versa, but in any event here is what the SST and SWCF anomalies from LENS look like for the period 1950-1979. It seems the midlatitudes are primarily responsible for the anomalous warmth during this period.
![temp_TS](https://github.com/user-attachments/assets/802242f3-fd14-488c-8782-77c84380f549)
![temp_SWCF](https://github.com/user-attachments/assets/58d7e2d3-35a6-45a5-8c22-0407c9dbdbd6)

**Reply by adamrher** [reply]
Julio just pointed out to me that my global diff's of TS (top left corner) indicate 98b/104 are cooler than LENS. Cecile's time-series has this time period as warmer than LENS. Is that because the time-series are anomalies from the beginning of the run? Would it be useful to look at non-normalized global TS vs. LENS time-series?

**Comment by islasimpson** [comment]
Here's a look at the F-cases.  I think it's a little too early to tell.  I'll try to update over the weekend before the machine goes down.

The following are anomalies relative to the 1900-1910 average (except where they're not anomalies)...

<img width="908" alt="Screen Shot 2024-09-13 at 2 53 06 PM" src="https://github.com/user-attachments/assets/de7bd241-76d4-45c2-ab7e-52084675b1af">

<img width="1198" alt="Screen Shot 2024-09-13 at 2 53 56 PM" src="https://github.com/user-attachments/assets/85fb90fd-6ec4-418d-891e-53544bfd6c40">

<img width="1159" alt="Screen Shot 2024-09-13 at 2 54 21 PM" src="https://github.com/user-attachments/assets/c1a9f347-2576-44d8-bdba-b68b88758db1">

**Reply by tilmes** [reply]
@islasimpson what is the difference between F_104 and F_103?

**Reply by islasimpson** [reply]
Everything is the same except the c_8 tuning.  103 doesn't have the additional tuning which made CDNUMC higher in 104.

**Reply by islasimpson** [reply]
Here are updated versions of the figures.  I think it's probably long enough now to say that the high CDNUMC might be a red herring.  

SW cloud forcing does not appear to be decreasing anymore in 103 than 104.  Note - the agreement with LENS here looks better because I've defined anomalies relative to the 1900-1950 period.

![Screenshot from 2024-09-15 15-28-48](https://github.com/user-attachments/assets/aa6e4be6-0dcd-49ab-b316-7bd14ff1154d)

The cloud liquid it not increasing in the way it does in LENS2.

![Screenshot from 2024-09-15 15-30-15](https://github.com/user-attachments/assets/d2458486-60fe-4d96-b698-486ce6250495)

Neither is the CDNUMC.  I think the CDNUMC plot really tells us that it doesn't matter where CDNUMC starts, it's not increasing in the same way as it is in LENS2.  

![Screenshot from 2024-09-15 15-32-13](https://github.com/user-attachments/assets/9c5504e8-c85c-44f0-a51c-a8aa69d92128)

So, I think can perhaps forget about the baseline CDNUMC as being a key factor in governing how the shortwave cloud forcing is behaving over the 20th century.

**Reply by islasimpson** [reply]
Here's being a bit more quantitative about that.  This is showing 1900-1945 trend in CDNUMC versus 1900-1915 climatology.

![Screenshot from 2024-09-15 16-12-31](https://github.com/user-attachments/assets/2522cda6-6720-48c3-9606-45978237c30e)

**Reply by islasimpson** [reply]
Here are some scatter plots.  They're using 20 year running means centered on 1910-1975 but note that the new AMIPs don't extend that far.  All points are anomalies relative to 1900-1915.  Trying to get at the question of whether the SWCF may be responding to the temperature rise or causing the temperature rise.  Top left shows SWCF vs TS.  So, what this says that at a given increase in global mean temperature, the SWCF tends to be closer to zero in all the new runs than it is in the majority of LENS members.  So, when we condition on temperature anomalies, we still see that SWCF in LENS2 is more negative during this time period than it is in the newer runs. 

SWCF is highly correlated with AOD and it declines more at a given AOD than it does in the new runs (although the F-cases haven't reached the really high AOD yet.  

![Screenshot from 2024-09-15 17-04-45](https://github.com/user-attachments/assets/9be4bee2-7187-481f-9e83-fe070f9927f0)


The next plot is showing scatter plots of trends for 1900-1960.  Left one shows that across LENS2 members, the TS trend is correlated with the SWCF trend.  A member that has a relatively more negative SWCF trend has a weaker temperature trend.  104 looks like it could fit this LENS2 relationship.  It's characterized by a relatively less negative SWCF trend and warms more.  The second one shows SWCF versus AOD.  It also looks like there's a correlation here that I don't really understand.  AOD is rising in all members and SWCF is declining.  But for the inter-member spread, it's those that are characterized by a lower trend in AOD that are characterized by a more negative SWCF.  Maybe someone else understands this better?     

![Screenshot from 2024-09-15 17-16-35](https://github.com/user-attachments/assets/529b5d30-37cd-49c7-9235-0ac983c384ba)

**Reply by adamrher** [reply]
Hi Isla, I'm finally taking a look at this. Thanks for all this analysis, I agree with you that the CDNUMC baseline is probably not impacting the cloud feedbacks in a way that explains our warmer than usual post-war period.

> AOD is rising in all members and SWCF is declining. But for the inter-member spread, it's those that are characterized by a lower trend in AOD that are characterized by a more negative SWCF. Maybe someone else understands this better?

I don't know that I understand this any better, but I suspect AODVIS is not the same as cloud nucleation, and I do know that most a of the aerosol burden is not cloud borne (roughly 1% of the aerosols are activating). I also wonder if the spread in the AODVIS trends are basically epsilon-like noise, and that temperature differences are larger and may be more responsible for inter-member spread.
